src/common/config/config-loader.service.spec.ts

```diff
@@ -15,6 +15,8 @@ const DEFAULTS = {
   RABBITMQ_URL: 'some-rabbit-url',
   RABBITMQ_LOGIN: 'some-rabbit-login',
   KEYS_API_URL: 'keys-api',
+  DATA_BUS_ADDRESS: 'DATA_BUS_ADDRESS',
+  DATA_BUS_PROVIDER_URL: 'DATA_BUS_PROVIDER_URL',
```

Comment:
Consider adding this on council daemons side as well. It could be multiply rpc for one chain, or for different. Since the API is the same and we can deploy the contract with the same address on different chains this kind of fallback seems easy enough to implement.

Reply:
We've discussed this before and you suggested not to do it.